lib/text_field.js

```diff
@@ -173,6 +171,7 @@ var TextField = (function (_Input) {
   }, {
     key: '_textDidChange',
     value: function _textDidChange() {
+      debugger;
```

Comment:
ewww.. I thought I stopped that, maybe I got the `src` and not `lib`. Nice find.